src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,91 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+// Increase the count after successful form submission
+// to reset touched status of all the `Field`s
+const initialValue = {
+  title: '',
+  description: '',
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+type Props = {
+  onAdd:(movie:Movie) => void;
+};
+
+export const NewMovie:React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [textAtributes, setTextAtributes] = useState(initialValue);
+
+  const addOne = () => {
+    setCount(current => current + 1);
+  };
+
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  } = textAtributes;
+
+  const disabled = title.trim() && imgUrl.trim()
```

Comment:
The current check is set to succeed, so the name `disabled` is a bit confusing
```suggestion
  const isActive = title.trim() && imgUrl.trim()
```